### DISCUSSION TITLE
Http Proxy Env Setting

### DISCUSSION THREAD

**Discussion by Stillrainy** [aio-libs/aiohttp#5855] [Q&A] [answered]
Can I set the http proxy for aiohttp module globally? Just like set proxy for git: 
<pre><code>git config --global --add http.proxy ...</code></pre>

Actually, I am trying to set the proxy for another Python program which mainly using aiohttp to download files. Setting such environment key is an easier way for me. Perhaps, is there has any alternative solution for me?

**Comment by webknjaz** [comment, marked as answer]
If you set `trust_env=True` in your `ClientSession` instance, it'll automatically use the values of the standard `HTTP_PROXY`/`HTTPS_PROXY` env vars and the credentials from `~/.netrc`: https://docs.aiohttp.org/en/stable/client_reference.html#aiohttp.ClientSession.trust_env.
